integration-tests/jakarta-ee/pom.xml

```diff
@@ -407,7 +408,7 @@
                         <phase>pre-integration-test</phase>
                         <configuration>
                             <executable>sed</executable>
-                            <commandlineArgs>-i ${mac-backup-file} -e 's/jvm-options&gt;\[17|/jvm-options&gt;\[16|/g'
+                            <commandlineArgs>-i ${mac-backup-file} -e 's/jvm-options>\[17|/jvm-options>\[16|/g'
```

Comment:
For anyone else wondering - this is valid XML, only `<` and `&` should be escaped in XML.